src/ResourceManager/Network/Commands.Network/ApplicationGateway/FirewallConfiguration/NewAzureApplicationGatewayWebApplicationFirewallConfigurationCommand.cs

```diff
@@ -0,0 +1,36 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Network.Models;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Network
+{
+    [Cmdlet(VerbsCommon.New, "AzureRmApplicationGatewayWebApplicationFirewallConfiguration"), OutputType(typeof(PSApplicationGatewayWebApplicationFirewallConfiguration))]
```

Comment:
@allanbomsft cmdlets that make any changes or have side effects should implement `ShouldProcess`. You can read more about it [here](https://gist.github.com/markcowl/338e16fe5c8bbf195aff9f8af0db585d).